nucleus/pom.xml

```diff
@@ -255,13 +262,43 @@
                 </plugin>
 
                 <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>${maven.surefire.plugin.version}</version>
                     <configuration>
                         <trimStackTrace>false</trimStackTrace>
+                        <useSystemClassLoader>true</useSystemClassLoader>
+                        <excludes>
+                           <exclude>**/*ITest.java</exclude>
+                        </excludes>
                     </configuration>
                 </plugin>
+                <plugin>
```

Comment:
Separated integration tests, gives possibility separate simple quick unit tests and those larger.